### ISSUE TITLE
Examples for multiclass text classification

### ISSUE THREAD

**Issue by stenpiren** [haifengl/smile#567]
Is it possible to include some examples of how to do multiclass text classification using the SMILE library? While the library is great, I feel it lacks meaningful tutorials in Java on how to actually use it.

**Comment by haifengl** [comment]
We will add more tutorials.

**Comment by stenpiren** [comment]
> We will add more tutorials.

Thanks. That would be great. I also find it hard to understand current tutorials in the website for Java, where it says "var variableName" etc... the syntax isn't Java. Seems hard to follow it for newbies. 

Have you thought of doing video tutorials or more robust tutorials with different examples? Could be first normal regression, then classification taking both numerical data, text data and also image data. It gives a "spark" into the functionalities of the library and potential more people can be interested in using it and contributing.

**Comment by mzeimet** [comment]
why would "var variableName" not be Java syntax?

**Comment by stenpiren** [comment]
> why would "var variableName" not be Java syntax?

It should be more specific like type of variable for instance.

**Comment by haifengl** [comment]
It is Java 9+ syntax.

**Comment by haifengl** [comment]
We add an example to Quick Start page.
